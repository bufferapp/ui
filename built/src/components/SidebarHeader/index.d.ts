export { default } from './SidebarHeader';
//# sourceMappingURL=index.d.ts.map