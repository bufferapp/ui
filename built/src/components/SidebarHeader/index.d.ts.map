{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../../../../src/components/SidebarHeader/index.ts"],"names":[],"mappings":"AAAA,OAAO,EAAE,OAAO,EAAE,MAAM,iBAAiB,CAAA"}