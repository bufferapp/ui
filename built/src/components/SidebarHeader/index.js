export { default } from './SidebarHeader';
//# sourceMappingURL=index.js.map