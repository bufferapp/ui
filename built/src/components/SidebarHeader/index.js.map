{"version":3,"file":"index.js","sourceRoot":"","sources":["../../../../src/components/SidebarHeader/index.ts"],"names":[],"mappings":"AAAA,OAAO,EAAE,OAAO,EAAE,MAAM,iBAAiB,CAAA","sourcesContent":["export { default } from './SidebarHeader'\n"]}