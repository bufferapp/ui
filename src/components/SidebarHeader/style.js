import styled from 'styled-components'
import { grayLight } from '../style/colors'

export const TitleContainer = styled.div`
  display: flex;
  align-items: center;
  justify-content: space-between;
  padding-right: 8px;
  padding-bottom: 8px;
  border-bottom: 1px solid ${grayLight};
`
