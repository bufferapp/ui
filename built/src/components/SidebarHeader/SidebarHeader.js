import React from 'react';
import PropTypes from 'prop-types';
import { TitleContainer } from './style';
import Text from '../Text/Text';
const SidebarHeader = ({ title }) => (React.createElement(TitleContainer, null,
    React.createElement(Text, { type: "label" }, title)));
SidebarHeader.propTypes = {
    title: PropTypes.string.isRequired,
};
export default SidebarHeader;
//# sourceMappingURL=SidebarHeader.js.map