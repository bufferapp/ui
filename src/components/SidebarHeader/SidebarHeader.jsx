import React from 'react'

import PropTypes from 'prop-types'
import { TitleContainer } from './style'
import Text from '../Text/Text'

const SidebarHeader = ({ title }) => (
  <TitleContainer>
    <Text type="label">{title}</Text>
  </TitleContainer>
)

SidebarHeader.propTypes = {
  title: PropTypes.string.isRequired,
}

export default SidebarHeader
