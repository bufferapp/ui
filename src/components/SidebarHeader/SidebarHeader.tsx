import React from 'react'

import PropTypes from 'prop-types'
import { TitleContainer } from './style'
import Text from '../Text/Text'

// @ts-expect-error TS(7031) FIXME: Binding element 'title' implicitly has an 'any' ty... Remove this comment to see the full error message
const SidebarHeader = ({ title }) => (
  <TitleContainer>
    <Text type="label">{title}</Text>
  </TitleContainer>
)

SidebarHeader.propTypes = {
  title: PropTypes.string.isRequired,
}

export default SidebarHeader
