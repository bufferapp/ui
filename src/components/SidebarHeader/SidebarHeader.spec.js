/* eslint-disable react/jsx-filename-extension */
import snap from 'jest-auto-snapshots'
import 'jest-styled-components'

import SidebarHeader from './SidebarHeader'

snap(SidebarHeader, './SidebarHeader.jsx')
