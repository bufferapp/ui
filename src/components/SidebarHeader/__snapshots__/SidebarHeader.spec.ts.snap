// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-auto-snapshots > SidebarHeader Matches snapshot when passed only required props 1`] = `
<styled.div>
  <Text
    color="grayDarker"
    hasError={false}
    type="label"
  >
    jest-auto-snapshots String Fixture
  </Text>
</styled.div>
`;
